libs/transloco/src/lib/transloco.service.ts

```diff
@@ -790,7 +790,10 @@ export class TranslocoService implements OnDestroy {
 
   private getMappedScope(scope: string): string {
     const { scopeMapping = {} } = this.config;
-    return scopeMapping[scope] || toCamelCase(scope);
+    return (
+      scopeMapping[scope] ||
+      (this.config.scope.keepNamespaceCasing ? scope : toCamelCase(scope))
```

Comment:
We are already extracting values from the config in L792

Reply:
Done